File: mario.py
import cs50


while True:
    h = cs50.get_int("Height: ")
    if h > 8:
        print("Height Must Be No Greater Than 8")
    elif h <= 0:
        print("Height Must Be A Positive Integer")
    else:
        break

for x in range(h):
    print(" " * (h - x - 1), end="")
    print("#" * (x + 1), end="")
    print("  ", end="")
    print("#" * (x + 1))

File: mario.c
#include<cs50.h>
#include<stdio.h>

//int variable for pyramid height
//prompt reader if input is not an int or less than 1

int main(void)
{
    int n;
       do
{
    n = get_int("size:");
}
while ( n<1);
for (int i=0; i<n; i++)
{
    for (int j=0; j<n; j++)
    {
        if(i+j+1<n)
        {
            printf(" ");
        }
        else
            {
                printf("#");
            }



    }
        printf("\n");
}
}

